### ISSUE TITLE
How take into account for batch effect ?

### ISSUE THREAD

**Issue by lmassenet-regad** [neurorestore/Libra#3]
Hi, 

Thank you very much for this very useful package!

As written in your vignette :  " If batch effects are present in the data, these should be accounted for, e.g., using Seurat or Harmony, to avoid biasing differential expression by technical differences or batch effects." 
Thus, I am not sure that I understand :  how do you take into account for batch effect in the Libra package for differential analysis ? 

1- From a seurat object, what is the assay considered ("RNA" versus "integrated" after CCA integration for exemple) as input of run_de() function?

2- I would say that batch effect correction is important for cell_type annotation only,  in order to correctly label cells and add  information to metadata. Am I correct or am I missing something important?

I hope that you can help me clarifying this point, 
Thanks, 
Lucile

**Comment by juicejulia** [comment]
Hello Jordan,
I am wondering whether there is a way to add batch effects in Libra? Or do we have to use the pseudobulk matrices generated in LIbra as input in DESeq2, limma or edgeR? 
Thank you!

**Comment by jordansquair** [comment]
Yes we haven't implemented that within Libra. But you can just take the pseudobulk matrices from to_pseudobulk and go forward with your preferred method :).

**Comment by dcruz5h** [comment]
Regarding this, does  Libra::to_pseudobulk command allow to retrieve more than one label_col such as sex and it disease state ?

**Comment by jordansquair** [comment]
Right now just one label - we will work on updating this along with the potential for more complex design inputs to the pseudobulk DE.
